FEWD/Week03/lesson06/homework/startup_matchmaker/style/style.css

```diff
@@ -0,0 +1,243 @@
+* {
+  box-sizing: border-box;
+}
+
+.clearfix:before, .clearfix:after {
+  content: "";
+  display: table;
+}
+
+.clearfix:after {
+  clear: both;
+}
+
+/*HEADER, BODY, FOOTER OF PAGES*/
+
+body {
+  width: 100%;
+  font-family: "Source Sans Pro", helvetica, sans-serif;
+  color: #000;
+  font-size: 12px;
+  line-height: 1.4;
+}
+
+header {
+  color: white;
+  background-color: #000;
+  height: 75px;
+  border-bottom: 5px solid #ffff1a;
+}
+
+h1 {
+  font-size: 40px;
+  font-weight: 700;
+  margin-left: 25px;
+  float: left;
+  font-family: Merriweather;
+  padding-top: 5px;
+}
+
+nav {
+  float: right;
+  height: 75px;
+  padding: 20px 0;
+}
+
+
+header li {
+  display: inline-block;
+  font-size: 20px;
+  font-weight: 600;
+  padding-right: 25px;
+}
+
+.font a:hover {
+  background-color: #b3b3b3;
+  padding: 15px 10px;
+}
+
+header li a:hover {
+  background-color: #b3b3b3;
+  padding: 22px 10px 18px;
+}
+
+header li a {
+  text-decoration: none;
+  color: white;
+}
+
+.font {
+  font-size: 24px;
+  text-transform: uppercase;
+  font-weight: 700;
+  font-family: oswald;
+}
+
+footer {
+  background-color: #000;
+  color: white;
+  font-size: 14px;
+}
+
+footer p {
+  padding: 10px;
+}
+
+/*END OF GENERIC STYLING*/
+
+/*INDEX HTML CSS*/
+.tag-line {
+  background-color: #cccccc;
+  padding: 10px;
+  font-size: 28px;
+  font-style: italic;
+  font-weight: 600;
+  text-align: center;
+  font-family: Merriweather;
+}
+
+.strikethrough {
+  text-decoration: line-through;
+  color: #b3b3b3;
+}
+
+.underline {
+  text-decoration: underline;
+  -webkit-text-decoration-color: #ffff1a;
+  text-decoration-color: #ffff1a;
+}
+
+.image {
+  background: url(../images/coworking.jpg) no-repeat center top #b3b3b3;
+  background-size: cover;
+  height: 500px;
```

Comment:
Use padding to add some space between the `<section>` and the `<aside>`, so that the `<aside>` isn't sitting flush to the top-right of the image that you've set as the background here. Since the `<aside>` is floated to the right, you'll also need to apply the `clearfix` class to the `<section>` for this to work.